pkg/distro/rhel/rhel9/distro.go

```diff
@@ -346,6 +346,22 @@ func newDistro(name string, major, minor int) *rhel.Distribution {
 			},
 			mkEC2ImgTypeAarch64(),
 		)
+
+		// CVM is only available starting from 9.6
+		if common.VersionGreaterThanOrEqual(rd.OsVersion(), "9.6") {
```

Comment:
Do we really need this condition?

It is not like we will backport the image type to 9.6. What would be the problem if one builds 9.5 CVM image on 9.7? We can also restrict what is available on-prem and limiting our options for SaaS version seems unnecessary. Especially when this is not a change in behavior, partitioning, or configuration... :thinking:

Reply:
I think the uki kernel didn't exist in older versions.  Not entirely sure when it was added.

Reply:
9.6 is the first supported version. Where 9.4 is missing the kernel-virt-uki-addons package and 9.5 is missing FIPS and so on.

Reply:
ACK. It would be nice to add some additional context to the comment, mainly to ensure that we do not end up wondering in the future if we can't enable it for older RHEL-9 minor versions.